src/runtime/plan/README.md

```diff
@@ -0,0 +1,83 @@
+# Planning in the Shell
+
+By default plans are produced and consumed locally in the browser.
+When using [arcs-live](https://polymerlabs.github.io/arcs-live/shells/web-shell) by default both arcs and suggestions are serialized to Firebase.
+
+## URL params
+Suggestions storage can be controlled by **plannerStorage** url param. 
+
+To use volatile storage for suggestion go to:  
+https://polymerlabs.github.io/arcs-live/shells/web-shell?plannerStorage=volatile
+
+To use PouchDb storage for suggestion go to:  
+https://polymerlabs.github.io/arcs-live/shells/web-shell?plannerStorage=pouchdb://local/random-db-name  
+The suggestion values will be found in Chrome's inspector, under Application > Storage > Local Storage.
+
+
+To populate Arcs Extension strategy explorer, add the **plannerDebug** url param:  
+https://polymerlabs.github.io/arcs-live/shells/web-shell?plannerDebug
+
+# Remote Planning 
+To consume plans produced on the server, disable shell planning with an **onlyConsumer** url param, ie:  
```

Comment:
`onlyConsumer` is cited here but the example below uses `plannerOnlyConsumer`, which is accurate?

Reply:
fixed. i actually moved this part to the top sections with all the other links to arcs-live.